boot/freeldr/freeldr/ntldr/arch/i386/pae.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * PROJECT:     FreeLoader
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Physical Address Extension (PAE) paging mode definitions.
+ * COPYRIGHT:   Copyright 2022 Vadim Galyant <vgal@rambler.ru>
+ */
+
+#pragma once
+
+#define PAE_TABLES_START ((ULONG_PTR)0xC0000000)
```

Comment:
I looked at it again, and think that as we don't compile PAN/non-PAE version of freeloader separately (like we do with the kernel), these definitions should go to i386.h header. It is small, so that should be OK

Reply:
Hm.. Revert it? [[FREELDR] Moving the PAE-specific in a separate header file.]( https://github.com/reactos/reactos/pull/4272/commits/13be23c6005bdaff16f2b3a9dcab0e6cf09cc337)

Reply:
Indeed the ntldr part of freeldr should support both PAE and non-PAE at the same time.